aspnet/conceptual-overview/understanding-aspnet5-apps.rst

```diff
@@ -36,192 +29,142 @@ has been refactored into files that are generally smaller and more focused.
 
 .. image:: understanding-aspnet5-apps/_static/dependencies.png
 
-The bower dependencies are controlled by the *bower.json* file. You'll notice that each of the items listed in the figure above correspond to dependencies listed in bower.json:
+The bower dependencies are controlled by the *bower.json* files, located in each of the sub-folders of *wwwroot/lib*. You'll notice that each of the items listed in the figure above correspond to dependencies listed in the *bower.json* files:
 
-.. image:: understanding-aspnet5-apps/_static/bower-json.png
+.. image:: understanding-aspnet5-apps/_static/bower-files.png
 
-Each dependency is then further configured in its own section within the *bower.json* file, indicating how it should be deployed to the *wwwroot* folder when the bower task is executed.
-
-By default, the bower task is executed using gulp, which is configured in *gulpfile.js*. The current web template’s gulpfile includes tasks for copying and cleaning script and CSS files from the bower folder to a ``/lib`` folder in ``wwwroot``.
-
-.. image:: understanding-aspnet5-apps/_static/bower-npm-folders.png
+Dependency are then further configured in their own section within each *bower.json* file, indicating how it should be deployed to the *wwwroot* folder.
 
 .. _`server side dependencies` :
 
 Server Side Dependency Management
 ---------------------------------
 
-The *References* folder details the server-side references for the project. It should be familiar to ASP.NET developers, but it has been modified to differentiate between references for different framework targets, such as the full ASP.NET 5.0 vs. ASP.NET Core 5.0.  Within each framework target, you will find individual references, with icons indicating whether the reference is to an assembly, a NuGet package, or a project. Note that these dependencies are checked at compile time, with missing dependencies downloaded from the configured NuGet package source (specified under Options – NuGet Package Manager – Package Sources).
+The *References* folder, shown within **Solution Explorer** in Visual Studio, details the server-side references for the project. It should be familiar to ASP.NET developers, but it has been modified to differentiate between references for different framework targets, such as the full DNX 4.5.1 vs. DNX Core 5.0.  Within each framework target, you will find individual references, with icons indicating whether the reference is to an assembly, a NuGet package, or a project. Note that these dependencies are checked at compile time, with missing dependencies downloaded from the configured NuGet package source (specified under **Options** –> **NuGet Package Manager** –> **Package Sources**).
 
 .. image:: understanding-aspnet5-apps/_static/references.png
 
+For more information about NuGet, see `Introduction to NuGet <https://docs.asp.net/en/latest/conceptual-overview/nuget.html>`_.
```

Comment:
Since we don't have an Intro to NuGet doc yet let's instead link to https://docs.nuget.org.